Chapters/0-1-Prologue.md

```diff
@@ -0,0 +1,49 @@
+## Prologue
+
+<!--
+You cannot hear the first note of a ziyu's song. There is no grand entrance or catchy hook. No meandering conversation between a point and its counter. Each leaf is a gentle voice in a choir of the wind, finding its voice with the morning breeze. The tree is silent in the night, when the air is still and the stems curl up into themselves. But each morning, the light of Ara grows upon the horizon and awakens the breeze. You start to recognize the sound, only it's been there for a while, and you can't place exactly when it began. By the time Ara is full, every leaf is perched and singing, their harmonies intertwined in unimaginable complexity.
+
+It was under this tree that I was born. Among its leaves and stems and branches, I made my first cry to the world and the world sang back to me. My mother used to tell me how beautiful it was to hold me in have arms for the first time in this place. I still imagine it sometimes when I come here: waking up after a nap, hidden within the greatest sound in all the world. Thi would still be asleep, and I would stare at hae hair bobbing in the wind against the pale light of Ara.
+
+The traditions say it's good luck to be born under a ziyu, and to be born under its song is a blessing unmatched by any other. I've never felt particularly blessed. I was born a normal maman in a farming town in Heiko province. As a child I tilled the soil and harvested root vegetables. I once caught the milta that had nibbled through at least 50 of our pohua crop, so I guess I'm quick as well, but that's pretty typical for a maman. I tried my luck as an apprentice at the observatory, but fate decided that it was my mother's time to go. Hae farm, and all of the responsibilities and expectations that hae community imposed upon it, had fallen upon me.
+
+Once, I fell in love with a man and a woman that I doted on in a drunken stupor and I was so afraid of my feelings that I joined with them as soon as I could so I would never have to fall in love again. As my luck would have it, they make my life easier for the most part, but occasionally I think that I'd prefer the company of a screaming toddler, though I won't get into that right now. These days, I leave my adventures to the dealings of my farm, and my library of books.
+
+But this day was special. It was my annual tradition with the ziyu – of returning to my place of birth. Though my parents had passed, I still felt their presence whenever I came back. They would welcome me as the tree did with its great branch-arms stretched wide toward Ara. "I'm back," I'd say, and they breathed a sigh of relief that I had made it another year. I'd settle in against its trunk and then its song would start to lift.
+
```

Comment:
Did you mean to take this entire part out?

I liked the original prologue; it had a poetic essence.

Reply:
Yea. I went back to the original prologue.